pootle/static/js/editor/app.js

```diff
@@ -2219,6 +2219,11 @@ PTL.editor = {
     if (!sText.length || src === tgt) {
       return;
     }
+    // Exit if source language is not supported by tm service.
+    if (this.settings.tmSourceLanguages &&
```

Comment:
If `this.settings.tmSourceLanguages` is empty or undefined we also want to abort quickly.